source/_references/sourceimages.md

```diff
@@ -23,8 +23,8 @@ Source images are your original images you upload to Rokka. You should always se
 | size | Size of images in bytes |
 | width | Width of image in pixels |
 | height | Height of image in pixels |
-| dynamic_metadata | Can contain data that will alter the image identifying hash if altered |
-| user_metadata | Contains meta data set by the API user. Changing this will not alter the main hash identifying the image |
+| dynamic_metadata | Can contain data that will alter the image identifying hash if altered. See [dynamic metadata](dynamic-metadata.html) |
+| user_metadata | Contains custom meta data set by the API user for later searches. Changing this will not alter the main hash identifying the image. See [user metadata](user-metadata.html) |
```

Comment:
searching is the only purpose of user metadata, right?

Reply:
the user metadata will also returned, when you get the meta data of just one image via hash lookup. But maybe, for the enduser, that's the same as searching

Reply:
pushed a fix for this directly to master: ff014cd403ae34f33902eb65825839fad3a02104